pokemon_v2/tests.py

```diff
@@ -1,4 +1,6 @@
 import json
+
+from django.core.files.uploadedfile import SimpleUploadedFile
```

Comment:
Can you remove the unused import?

Reply:
just removed

Reply:
Awesome, once you run `make format`, I'll review it again and approve